include/deal.II/dofs/dof_tools.h

```diff
@@ -1657,6 +1657,20 @@ namespace DoFTools
   extract_locally_active_dofs(const DoFHandler<dim, spacedim> &dof_handler,
                               IndexSet &                       dof_set);
 
+  /**
+   * Same function as above but for a certain (multigrid-)level.
+   * This function returns all DoF indices that live on
+   * all locally owned cells (including on the interface to ghost cells) on the
+   * given level.
+   */
+  template <int dim, int spacedim>
+  void
+  extract_locally_active_level_dofs(
+    const DoFHandler<dim, spacedim> &dof_handler,
+    IndexSet &                       dof_set,
+    const unsigned int               level);
+
+
```

Comment:
Nit-picking, but may I ask you to remove the second empty line to be consistent with the whitespace we add between functions?